aws/resource_aws_cloudwatch_log_subscription_filter.go

```diff
@@ -187,6 +195,22 @@ func resourceAwsCloudwatchLogSubscriptionFilterDelete(d *schema.ResourceData, me
 	return nil
 }
 
+func resourceAwsCloudwatchLogSubscriptionFilterImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	idParts := strings.Split(d.Id(), "|")
+	if len(idParts) < 2 {
+		return nil, fmt.Errorf("unexpected format of ID (%q), expected <log-group-name>/<filter-name-prefix>", d.Id())
```

Comment:
This error should use `|` not `/` 😄 

```suggestion
		return nil, fmt.Errorf("unexpected format of ID (%q), expected <log-group-name>|<filter-name>", d.Id())
```